cmd/account/reset.go

```diff
@@ -44,19 +44,31 @@ func newCmdReset(streams genericclioptions.IOStreams, flags *genericclioptions.C
 		"Skip the prompt check")
 	resetCmd.Flags().BoolVar(&ops.resetLegalEntity, "reset-legalentity", false,
 		`This will wipe the legalEntity, claimLink and reused fields, allowing accounts to be used for different Legal Entities.`)
+	resetCmd.Flags().BoolVarP(&ops.editState, "edit-state", "e", false,
+		"Change only the state of an account.")
+	resetCmd.Flags().StringVar(&ops.stateInput, "state", "",
+		"Select the state to change to. Required when using the --edit-state flag. Valid values are Ready or Failed")
```

Comment:
Why do we need 2 parameters here? Wouldn't be setting a string enough to know we should set the state? (e.g. `-e failed`)